hashing/te-tag-query-java/APIExamples.java

```diff
@@ -0,0 +1,98 @@
+// Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved
+
+// ================================================================
+// The main reference tooling is com/facebook/threatexchange/TETagQuery.java,
+// which is likely most useful. However, this file is a more clear-cut example
+// of incorporating the API into your own framework.
+//
+// Compile with
+//
+//   javac com/facebook/threatexchange/*.java APIExamples.java
+//
+// and run with
+//
+//   java APIExamples
+// ================================================================
+
+import com.facebook.threatexchange.Net;
+import com.facebook.threatexchange.DescriptorPostParameters;
+
+import java.io.IOException;
+
+public class APIExamples {
+  // ----------------------------------------------------------------
+  public static void main(String[] args) throws IOException {
+    Net.setAppToken("TX_ACCESS_TOKEN");
+    submitExample();
+    updateExample();
+    copyExample();
+  }
+
+  // ----------------------------------------------------------------
+  private static void submitExample() {
```

Comment:
these examples are great! I think it might be worth adding more explanatory comments here and there for each example language e.g. 
"Add all required fields" 
"Privacy Type is either HAS_WHITELIST or HAS_PRIVACY_GROUP" 
"ID of descriptor to make a copy of"

Especially if we expect/want users to copy this. We might as well have them copy good documentation as well. :)

Reply:
nice